### PR TITLE
Write Tableau API output to debug files

### PR DESCRIPTION
**Pull request by mars-lan**
### 🤔 Why?

Log the raw API output to `log.zip` help debug issues.

### 🤓 What?

As titled.

### 🧪 Tested?

Manually ran against a production deployment and verified the output:

Screenshot 2022-12-05 at 6 01 48 AM

### CODE REVIEW

**Review by alyiwang — APPROVED** [review]

**Comment by alyiwang** [comment]
CI failed

**Comment by mars-lan** [comment]
> CI failed

Yeah need to improve test coverage.

File: metaphor/common/logger.py [hunk 1]
```diff
@@ -30,3 +32,13 @@ def get_logger() -> logging.Logger:
 
 def add_debug_file(file: str) -> None:
     debug_files.append(file)
+
+
+def json_dump_to_debug_file(value: Union[List, Dict], file_name: str) -> str:
```
Comment (elic-eon) [inline metaphor/common/logger.py:37]: Just use `Any` type for `value`?

**Review by alyiwang — APPROVED** [review]
LGTM

**Review by elic-eon — APPROVED** [review]

**Pull request merged**